inc/osvr/ClientKit/SkeletonC.h

```diff
@@ -0,0 +1,71 @@
+/** @file
+    @brief Header
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_SkeletonC_h_GUID_999D5F46_F1C3_41B5_0EEC_9746ACBA9270
+#define INCLUDED_SkeletonC_h_GUID_999D5F46_F1C3_41B5_0EEC_9746ACBA9270
+
+/* Internal Includes */
+#include <osvr/ClientKit/Export.h>
+#include <osvr/Util/APIBaseC.h>
+#include <osvr/Util/ReturnCodesC.h>
+#include <osvr/Util/ClientOpaqueTypesC.h>
+#include <osvr/Util/ClientReportTypesC.h>
+
+/* Library/third-party includes */
+/* none */
+
+/* Standard includes */
+/* none */
+
+OSVR_EXTERN_C_BEGIN
+
+OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrClientGetSkeletonBoneId(
+    OSVR_ClientInterface skeletonIface, const char *boneName,
+    OSVR_SkeletonBoneCount *boneId);
+
+OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrClientGetSkeletonBoneState(
+    OSVR_ClientInterface skeletonIface, OSVR_SkeletonBoneCount boneId,
+    OSVR_SkeletonBoneState *state);
+
+OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrClientGetSkeletonJointId(
+    OSVR_ClientInterface skeletonIface, const char *jointName,
+    OSVR_SkeletonJointCount *jointId);
```

Comment:
Same question here. Need a reverse to get the joint name of a given joint ID, and possibly a couple of methods to get all possible joint names.